codex/codex.nim

```diff
@@ -221,6 +224,7 @@ proc new*(T: type CodexServer, config: CodexConf, privateKey: CodexPrivateKey):
       .expect("Should start rest server!")
 
   switch.mount(network)
+
```

Comment:
Needed?

Reply:
It seems to be standard to have an ending newline, so I followed it.